automation/test.sh

```diff
@@ -324,6 +324,14 @@ spec:
 EOF
 fi
 
+# Enable jumbo-frames for a selected lane
+if [[ $TARGET == "k8s-1.19" ]]; then
+  echo "Enabling jumbo frames."
+  kubectl patch configmap/calico-config -n kube-system --type merge \
+    -p '{"data":{"veth_mtu": "8000"}}'
+
+  kubectl rollout restart daemonset calico-node -n kube-system
```

Comment:
https://docs.projectcalico.org/networking/mtu

Not sure doing this on the `kubevirtci` is something we want to do. Inclined to think it isn't.

Reply:
My religion tells me to keep all cluster preparations in kubevirtci. Ideally, kubevirt tests should be unaware of what they are running on. Even though you match the TARGET here, it is not "k8s-1.19-calico".